src/gc/gc.h

```diff
@@ -267,10 +267,6 @@ class IGCHeapInternal : public IGCHeap {
 
 protected: 
```

Comment:
You can delete this too.

Reply:
yeah you can delete both lines

`protected: `
`public:`

I think it was a merge error that they are both there.

Reply:
Done.